outlook/src/utils/Translator.ts

```diff
@@ -0,0 +1,78 @@
+//Threshold in miliseconds after which translations are considered as expired
+const EXPIRATION_TRESHOLD = 24 * 60 * 60 * 1000; //set to 24 hours
```

Comment:
Why 24 hours ?

Reply:
This is really subjective, based on the fact that these values won't change a lot (only when the user migrates to a new version and that new version introduces some changes there) and since it's highly unlikely that the user would want to change his language too often, a delay of 24 hours seems reasonable.